cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -203,7 +204,7 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
 
                 packageDownloadDirs.forEach { (pkg, dir) ->
                     try {
-                        Downloader.download(pkg, dir, allowMovingRevisions)
+                        Downloader(DownloaderConfiguration()).download(pkg, dir, allowMovingRevisions)
```

Comment:
Uhm, shouldn't this use the config from `ort.conf` instead of creating a default instance? Same below.

Reply:
Similar in other places below.